service/efs_submission_service.go

```diff
@@ -3,18 +3,41 @@ package service
 import (
 	"fmt"
 	"net/http"
+	"regexp"
 
+	"github.com/companieshouse/chs.go/log"
 	"github.com/companieshouse/go-sdk-manager/manager"
+	"github.com/companieshouse/insolvency-api/config"
 )
 
-// IsUserOnEfsAllowList uses the sdk to call the EFS api and return a boolean depending on whether or not the email address is on the allow list
+// IsUserOnEfsAllowList uses the sdk to call the EFS api and return a boolean depending on whether or not the email
+// address is on the allow list
 func IsUserOnEfsAllowList(emailAddress string, req *http.Request) (bool, error) {
 	api, err := manager.GetInternalSDK(req)
 	if err != nil {
 		return false, fmt.Errorf("error creating SDK to call transaction api: [%v]", err.Error())
 	}
 
+	// Get environment config - only required whilst feature flag to disable EFS lookup exists
+	cfg, err := config.Get()
+	if err != nil {
+		return false, fmt.Errorf("error configuring service: %s. Exiting", err.Error())
```

Comment:
New convention is to use `%w` to wrap errors
```suggestion
		return false, fmt.Errorf("error configuring service: %w. Exiting", err)
```